app/serializers/api/v1/project_serializer.rb

```diff
@@ -41,6 +41,18 @@ class Api::V1::ProjectSerializer
     project.commontator_thread.is_closed?
   end
 
+  # :nocov:
+  if Flipper.enabled?(:active_storage_s3)
```

Comment:
@VaibhavUpreti  Since this condition is on class level, application reboot is required after changing this flag